taskcluster/ac_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,172 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+
+from mozilla_version.maven import MavenVersion
+from taskgraph.actions.registry import register_callback_action
+
+from taskgraph.util.taskcluster import get_artifact
+from taskgraph.taskgraph import TaskGraph
+from taskgraph.decision import taskgraph_decision
+from taskgraph.parameters import Parameters
+from taskgraph.util.taskgraph import find_decision_task, find_existing_tasks_from_previous_kinds
+
+RELEASE_PROMOTION_PROJECTS = (
+    "https://github.com/mozilla-mobile/android-components",
```

Comment:
Bug: We have to add `"https://github.com/escapewindow/android-components"` for the initial end-to-end testing. Otherwise `actions.json` won't contain the `release-promotion` action. See https://firefox-ci-tc.services.mozilla.com/tasks/YVymMAawR56yGmBB35ZerQ#artifacts, for instance